components/select/index.tsx

```diff
@@ -20,6 +20,15 @@ export interface LabeledValue {
 
 export type SelectValue = RawValue | RawValue[] | LabeledValue | LabeledValue[];
 
+export interface OptionProps {
+  disabled?: boolean;
+  value?: string | number;
+  title?: string;
+  children?: React.ReactNode;
+  className?: string;
+  style?: React.CSSProperties;
+}
```

Comment:
Try https://codesandbox.io/s/dry-sea-y6mv7

Reply:
or

```jsx
import { OptionProps } from 'rc-select';

export OptionProps;
}
```

Reply:
@afc163  Could you please explain what you want me to do by performing these operations mentioned above?

Reply:
I tried that code sandbox link and **add import { OptionProps } from 'rc-select';** and after import, it gives this warning :
Module '"../node_modules/rc-select/lib"' has no exported member 'OptionProps'. Did you mean to use 'import OptionProps from "../node_modules/rc-select/lib"' instead?

So I think after merging this PR it will solve the above warning.

Reply:
Sorry, I mean `import { OptionProps } from 'rc-select/lib/Option';`.

Reply:
FYI, does https://codesandbox.io/s/dry-sea-y6mv7 help?

Reply:
@afc163 So you want me to add this import statement in that index.TSX file and export it from there instead of declaring an another interface in the same file right?

Reply:
Correct.

Reply:
So is there any need for creating PR for that?

Reply:
I offer your two solutions:

1. https://codesandbox.io/s/dry-sea-y6mv7 If this works, maybe we don't need this PR.
2. Or we should stick to this PR, but change content:

```diff
- export interface OptionProps {
-   disabled?: boolean;
-   value?: string | number;
-    title?: string;
-   children?: React.ReactNode;
-   className?: string;
-   style?: React.CSSProperties;
- }
+ import { OptionProps } from 'rc-select/lib/Option';
+ export OptionProps;
```

Reply:
No first one is not working for me but can we export OptionProps like this because it is a type?

Reply:
How about https://codesandbox.io/s/cold-flower-yeh7f?file=/src/App.tsx

Reply:
```
export type OptionProps;
```